Features/Lookuptable/README.md

```diff
@@ -0,0 +1,57 @@
+## Commcare lookuptable Test Script
+
+These tests ensure that the [lookup table ](https://confluence.dimagi.com/display/commcarepublic/Lookup+Tables)features work as expected and that there are no regressions
+The automated tests comprises of [these Lookuptable functionality.](https://docs.google.com/spreadsheets/d/1aTBv4zGeVuNTdnCyuleZbbpqOhPEGl0f78rcKAGWQHQ/edit#gid=1360466241)
+## Executing Scripts
+
+### <ins> On Local Machine </ins>
+
+#### Setting up test environment
+
+```sh
+
+# create and activate a virtualenv using your preferred method. Example:
+python -m venv venv
+source venv/bin/activate
+
+
+# install requirements
+pip install -r requires.txt
+
+```
+
+[More on setting up virtual environments](https://confluence.dimagi.com/display/GTD/QA+and+Python+Virtual+Environments)
+
+
+#### Running Tests
+
+
+ -   Copy `settings-sample.cfg` to `settings.cfg` and populate `settings.cfg` for
+the environment you want to test.
+- Run tests using pytest command like:
+
+```sh
+
+# To execute all the test cases 
+pytest -v --rootdir= Lookuptable/testCases
```

Comment:
path needs to be updated @sameena1415

Reply:
https://github.com/dimagi/dimagi-qa/pull/169/commits/efe548901636422a013ce3da7fa22bb9c0f6a3b1

Reply:
Pushed the changes. Commit code for reference: efe548901636422a013ce3da7fa22bb9c0f6a3b1